javascript/display-data.js

```diff
@@ -1,26 +1,46 @@
 //TODO: write the display functions to render the page
 var collection = [];
 
-var data = function(param){
+var Data = function(param){
   this.library = param.library;
   this.libraryURL = param.libraryURL;
   this.category = param.category;
   this.dateTested = param.dateTested;
   this.review = param.review;
 }
 
-data.prototype.display = function(){
-  $('#library-name').append(this.library);
-  $('#library-url').attr('href',this.libraryURL);
-  $('#library-url').append(this.libraryURL);
-  $('#content-review-template').append('<hr>');
+Data.prototype.display = function(){
+  var $template = $('article.template').clone();
+  $template.attr('data-category',this.category).css('color','white');
+  $template.find('h1').css('margin','0px');
```

Comment:
This was used during the initial stage of dynamically creating the element,
I'll clean this up in the next PR, tested that CSS still applies even on
elements that were created after the document successfully loaded

On Tue, Jun 7, 2016 at 6:19 PM, meganwalter <notifications@github.com>
wrote:

> In javascript/display-data.js
> <https://github.com/wortleyjk-codefellows/301n2-portfolio/pull/2#discussion_r66180247>
> :
>
> >    this.library = param.library;
> >    this.libraryURL = param.libraryURL;
> >    this.category = param.category;
> >    this.dateTested = param.dateTested;
> >    this.review = param.review;
> >  }
> >
> > -data.prototype.display = function(){
> > -  $('#library-name').append(this.library);
> > -  $('#library-url').attr('href',this.libraryURL);
> > -  $('#library-url').append(this.libraryURL);
> > -  $('#content-review-template').append('<hr>');
> > +Data.prototype.display = function(){
> > +  var $template = $('article.template').clone();
> > +  $template.attr('data-category',this.category).css('color','white');
> > +  $template.find('h1').css('margin','0px');
>
> Not a best practice to write CSS inline, wondering if there was a specific
> reason behind it?
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub
> <https://github.com/wortleyjk-codefellows/301n2-portfolio/pull/2/files/648615446c00d5c6f58c47f0cdfd4231b82910e0#r66180247>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ARVlClAy5Hi5d2TxKlk1i-WJWU-v1qmlks5qJhi4gaJpZM4IunF4>
> .
>